### ISSUE TITLE
Builder has private access

### ISSUE THREAD

**Issue by eliasbagley** [miguelbcr/RxPaparazzo#68]
Hey, I'm trying to change the file provider name, and the docs say to use `RxPaparazzo.Builder.setFileProviderAuthority(String authority)`, which has private access in the `RxPaparazzo` class. How can I do this?

**Comment by miguelbcr** [comment]
Hi @eliasbagley ,
it's public, you can see in [RxPaparazzo class](https://github.com/miguelbcr/RxPaparazzo/blob/2.x/rx_paparazzo/src/main/java/com/miguelbcr/ui/rx_paparazzo2/RxPaparazzo.java#L94) . 

`Builder` was refering to `single()` or `multiple()`. Here you have an example:

```
RxPaparazzo.single(fragment)
.setFileProviderAuthority("myAuthority")
.setFileProviderPath("myPath")
.size(new ScreenSize())
.usingCamera()
.subscribe(response -> {
   // Do your stuff
})
```

**Comment by eliasbagley** [comment]
Ah I see, I didn't realize it was something you needed to set each time. I was expecting there to be some sort of builder for `RxPaparazzo.register()`. It would be nice to have the ability to set a default file provider path other than "RxPaparazzo"

**Comment by miguelbcr** [comment]
Hi @eliasbagley !
sorry for the late.
I've released [version 0.5.3-2.x](https://github.com/miguelbcr/RxPaparazzo/releases/tag/0.5.3-2.x)
Let me know if it works for you
